src/test/java/ibm/jceplus/junit/openjceplusfips/TestRSA.java

```diff
@@ -8,11 +8,17 @@
 
 package ibm.jceplus.junit.openjceplusfips;
 
+import java.security.NoSuchAlgorithmException;
```

Comment:
Why do we even need this test if RSA is not supported or registered in FIPS mode?

Reply:
Should we remove `TestRSA_1024`, `TestRSA_2048` and `TestRSA_4096` that subclass `TestRSA` with different key sizes? I don't think they serve any purpose given the changes.

Reply:
I was just running single tests. After running the whole test after eliminating these three tests, there do many references related to TestRSA_1024, TestRSA_2048, TestRSA_4096. But since we changed the upper class, those references should also be removed. Because we dont need to run them triple times.

Reply:
But for the multithread class such as TestRSA_2048, should we keep it and test it in multithread? after changing its upper class from BaseTestRSA to BaseTest?

Reply:
Hello, Yes for testing the OpenJCEPlus provider we still want to run multi-threaded testing for RSA with various key sizes. This is still needed and that algorithm is still in use. For OpenJCEPlusFIPS provider, I do not believe that we need to keep the multi-threaded testing for the RSA Cipher test. Since that test is simply testing that the RSA Cipher is not being registered as an algorithm, there is no value in testing that path in a multi-thread way.

Reply:
okay, sound good to me, will remove all the references only for the OpenJCEPlusFIPS provider.

Reply:
TestRSA_1024, TestRSA_2048 andg TestRSA_4096 are removed. TestRSA_2048 in multithread is also removed.